### ISSUE TITLE
Progressive loading: slide gets attached outside the slideshow container

### ISSUE THREAD

**Issue by EmilEriksen** [malsup/cycle2#422]
Hi,

I think I've found a bug. With the following markup http://jsfiddle.net/34YNA/3/ it seems like the slides get attached outside the .cycle-slideshow div. I have no idea what causes it.

**Comment by albell** [comment]
Yes, this is a bug with progressive loading. Note also that a second progressive slide inside the custom script block never gets attached at all:

http://jsfiddle.net/SLquk/

**Comment by EmilEriksen** [comment]
I found out what is causing it; an image must be a direct child to the slide container. See the this markup:

http://jsfiddle.net/SLquk/1/

**Comment by albell** [comment]
Ok, glad you fixed it, but that's still a bug. A "progressive" slide should be able to consist entirely of non-img content. E.g. text, video, an iframe, etc. It shouldn't break when there are no imgs in a slide.
